From 3cf0b735db7806d4c1ee1fc67baf067ce33b7bf3 Mon Sep 17 00:00:00 2001
From: Uri Mashiach <uri.mashiach@compulab.co.il>
Date: Tue, 3 Sep 2019 16:20:08 +0300
Subject: [PATCH 04/10] sb-imx8x: rename node pca9555_sb

The CL-SOM-iMX8X module doesn't include an additional pca9555 node.

Rename the I2C GPIO extender node pca9555_sb to pca9555.

Signed-off-by: Uri Mashiach <uri.mashiach@compulab.co.il>
---
 arch/arm64/boot/dts/compulab/sb-imx8x.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/compulab/sb-imx8x.dts b/arch/arm64/boot/dts/compulab/sb-imx8x.dts
index 36f382a61478..d3e744203c5d 100644
--- a/arch/arm64/boot/dts/compulab/sb-imx8x.dts
+++ b/arch/arm64/boot/dts/compulab/sb-imx8x.dts
@@ -37,7 +37,7 @@
 		pagesize = <16>;
 	};
 
-	pca9555_sb: pca9555_sb@20 {
+	pca9555: pca9555@20 {
 		compatible = "nxp,pca9555";
 		gpio-controller;
 		#gpio-cells = <2>;
-- 
2.11.0

